scripts/scripts.js

```diff
@@ -305,6 +306,15 @@ async function loadLazy(doc) {
   }
   loadCSS(`${window.hlx.codeBasePath}/styles/lazy-styles.css`);
   loadFonts();
+
+  document.querySelectorAll('a').forEach((aElement) => {
+    if (!aElement.hash) {
+      return;
+    }
+    aElement.addEventListener('click', (evt) => {
+      removeHashTags(evt.target);
+    });
+  });
```

Comment:
I am not entirely sure if this is the right place to put this event listener. I am open to any suggestion if this needs to be moved somewhere else.